app/controllers/v3/isolation_segments_controller.rb

```diff
@@ -56,6 +60,9 @@ def destroy
     isolation_segment_model = IsolationSegmentModel.where(guid: params[:guid]).first
     resource_not_found!(:isolation_segment) unless isolation_segment_model
 
+    method_not_allowed!('DELETE', "the #{isolation_segment_model.name} Isolation Segment") if
```

Comment:
@SocalNick I don't think so - a 409 implies that you could tweak the request - but same method, same resource - and have it succeed.  That isn't the case here, there is literally no way to delete (or update) this particular segment.